bson/src/main/org/bson/codecs/CollectionCodec.java

```diff
@@ -47,7 +44,7 @@
  */
 @SuppressWarnings("rawtypes")
 final class CollectionCodec<C extends Collection<Object>> extends AbstractCollectionCodec<Object, C>
-        implements OverridableUuidRepresentationCodec<C>, Parameterizable {
+        implements OverridableUuidRepresentationCodec<C> {
```

Comment:
I noticed that while `CollectionCodec` implements `OverridableUuidRepresentationCodec`, `ParameterizedCollectionCodec` does not. The same goes about `MapCodecV2` and `ParameterizedMapCodec`. Is this intentional / OK?